cluster/cluster_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/obolnetwork/charon/testutil"
 )
 
-//go:generate go test . -v -clean -update
```

Comment:
`-clean` deletes everything inside the `testdata` dir including the versioned definition and lock json files